<ui version="4.0" >
 <class>InputTextDialog</class>
 <widget class="QDialog" name="InputTextDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Input Text</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTextEdit" name="inputText" >
       <property name="toolTip" >
        <string>Text to paste into the MSX</string>
       </property>
       <property name="acceptRichText" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="sendButton" >
         <property name="text" >
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton" >
         <property name="text" >
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>431</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
